apps/scraper/models.py

```diff
@@ -0,0 +1,10 @@
+from django.db import models
+
+
+class Site(models.Model):
+    title = models.CharField(max_length=50, verbose_name='Title')
+    main_url = models.CharField(verbose_name='Main url', max_length=50)
+    slug = models.CharField(verbose_name='Pseudo slug', max_length=100)
```

Comment:
Maybe SlugField can be more useful: https://docs.djangoproject.com/en/2.2/ref/models/fields/#slugfield

Reply:
> Maybe SlugField can be more useful: https://docs.djangoproject.com/en/2.2/ref/models/fields/#slugfield

No. I try that it. SlugField don't validate. Only we hit in slag last part of url, but that useless.